src/libraries/TransientStateLibrary.sol

```diff
@@ -24,10 +24,9 @@ library TransientStateLibrary {
     /// @dev returns 0 if the reserves are not synced
     /// @dev returns type(uint256).max if the reserves are synced but the value is 0
     function getReserves(IPoolManager manager, Currency currency) internal view returns (uint256) {
-        bytes32 slot = RESERVES_OF_SLOT;
         bytes32 key;
         assembly {
```

Comment:
shouldnt this be memory safe too?

Reply:
i see a few others in this contract too

Reply:
weirdly enough this compiles without it being memory safe